app/src/main/java/com/andryoga/safebox/data/db/entity/UserDetailsEntity.kt

```diff
@@ -9,7 +10,7 @@ data class UserDetailsEntity(
     @PrimaryKey(autoGenerate = true)
     val key: Int,
     val password: String,
-    val hint: String?,
+    @ColumnInfo(name = "hint")val hint: String?,
```

Comment:
no need to add column hint here, this annotation is useful when you want variable name and column name to be different.
but here both are same, so no need to add this